src/utilities/createRouter.ts

```diff
@@ -130,6 +140,8 @@ export function createRouter<const T extends Routes>(routes: T, options: RouterO
   const notFoundRoute = getRejectionRoute('NotFound')
   const { route, updateRoute } = createCurrentRoute(notFoundRoute)
 
+  history.startListening()
```

Comment:
Should listening just be the default starting state?

Reply:
I went back and forth on that. I'm happy to move it in. I had the thought that we might for some reason want to delay starting until 'install' or something but no good reason to.

Reply:
I don't feel strongly. I'm just in favor of keeping createRouter as simple as possible. 

I do kinda like putting it in the install 🤔